spec/rubocop/cop/style/numeric_predicate_spec.rb

```diff
@@ -207,11 +207,22 @@ def m(foo)
     it 'registers an offense for checking if a number is zero' do
       expect_offense(<<~RUBY)
         number.zero?
-        ^^^^^^^^^^^^ Use `number == 0` instead of `number.zero?`.
+        ^^^^^^^^^^^^ Use `(number == 0)` instead of `number.zero?`.
       RUBY
 
       expect_correction(<<~RUBY)
-        number == 0
+        (number == 0)
```

Comment:
Parentheses should not be added when it is not needed.

Reply:
Made updates.